Demo/Sources/Assets/Icons.xcassets/Markets 1/finn/cabin.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "MarketTravelFhh.svg",
```

Comment:
old icons use `pdf` new tori icons use `pdf` too, these use `svg`
it needs to be unified